examples/playground/src/configuration.js

```diff
@@ -9,6 +9,12 @@ import * as MeshLayers from '@deck.gl/mesh-layers';
 import {COORDINATE_SYSTEM} from '@deck.gl/core';
 import GL from '@luma.gl/constants';
 
+import {registerLoaders} from '@loaders.gl/core';
```

Comment:
Why is this here? need to rebase to latest master?

Reply:
I did global search and couldn't find any places `CSVLoader` is registered.

Reply:
OK you moved it from app.js to here, that is good.